frontend/src/libs/components/pagination/pagination.tsx

```diff
@@ -143,6 +164,7 @@ const Pagination: React.FC<Properties> = ({
           <Dropdown
             options={createOptions([5, 10, 20, 50, 100])}
             defaultValue={createOption(pageSize)}
+            isValueCustomized
```

Comment:
this value will be always true in dropdown, was it supposed to be?

Reply:
The dropdown is styled so that the text of the selected option matches the filter styles on the Home Page. Here in the pagination, the styles are different and I can't override them with a passed class because the dropdown styles use nested selectors and the specificity is higher. I added this props to the pagination to remove the styling of the selected option and it inherits it from the pagination. 
In pagination this props will always be true, but in the rest of the application it will not be.